packages/clay-charts/src/Geomap.js

```diff
@@ -19,6 +19,8 @@ class Geomap extends Component {
 			return;
 		}
 
+		this._setupPolling();
```

Comment:
This should go to DataComponent, as ChartBase has no knowledge at all about polling